### DISCUSSION TITLE
I don't receive the data.

### DISCUSSION THREAD

**Discussion by jrubiano01** [innoveit/react-native-ble-manager#1181] [Q&A]
Good day, I'm trying to connect via Bluetooth to a scale to get the data, the integration works, the device connects according to the example of the App, but I don't see the data reflected or it's not receiving it, what do I need, I attach link of the scale that is being used as an example.

https://www.transtekcorp.com/remote-patient-monitoring/gbf-2008-b/

**Comment by enzo69006** [comment]
Hello, did u use the Buffer ?
